website/docs/en-US/popconfirm.md

```diff
@@ -40,6 +40,44 @@ You can customise Popconfirm like:
 ```
 :::
 
+### Trigger event
+Click the button to trigger the event
+
+:::demo
+
+```html
+<template>
+<el-popconfirm
+  confirmButtonText='Yes'
+  cancelButtonText='No'
+  icon="el-icon-info"
+  iconColor="red"
+  title="Are you sure to delete this?"
+  @confirm="confirmEvent"
+  @cancel="cancelEvent"       
+>
+<template #reference>
+  <el-button>Delete</el-button>
+  </template>
+</el-popconfirm>
+</template>
+
+<script>
+  export default {
+      methods: {
+          confirmEvent() {
+              console.log("confirm!");
+          },
+          cancelEvent() {
+              console.log("cancel!");
+          }
+      }
+  }
+</script>
```

Comment:
Can you keep the indent at 2 space instead of 4 space?

Reply:
Other than this I have nothing to comment about, please address that so that we can merge this. Plus: `export` keyword under `script` tag should not have indention IMHO.